tests/utilities/test_functions.py

```diff
@@ -0,0 +1,8 @@
+import functools
+
+from compas.utilities import binomial
+
+
+def test_binomial():
+    assert binomial(3, 1) == 3
+    assert binomial(21, 10) == 352716
```

Comment:
can you make sure to test the boundary cases?
for example, what if one or both of the number is zero?
see the geometry tests for setting up tests that are expected to fail.
also you can parametrize the input so the two assertions count as two tests...